Makefile

```diff
@@ -4,7 +4,7 @@ endif
 VERSION=$(shell $(PYTHON) setup.py --version 2>/dev/null)
 PYTHON_DEVELOP_ARGS=$(shell if ($(PYTHON) setup.py develop --help 2>/dev/null | grep -q '\-\-user'); then echo "--user"; else echo ""; fi)
 DESTDIR=/
-AVOCADO_DIRNAME=$(shell echo $${PWD\#\#*/})
+AVOCADO_DIRNAME=$(notdir ${PWD})
```

Comment:
notdir is not available on f31, any reason for not using basename?

Reply:
Oh, I wasn't aware of this, I was testing it just for f32 and f33. Thanks, I will have to find a different way. I didn't use the `basename` because IIUC it works differently in `Makefile`. https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html

Reply:
> Oh, I wasn't aware of this, I was testing it just for f32 and f33. Thanks, I will have to find a different way. I didn't use the `basename` because IIUC it works differently in `Makefile`. https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html

I guess @beraldoleal meant the executable `basename`, which is part of the `coreutils` package, and not the GNU Make function of the same name.

Reply:
Ok, my bad, I will force pushed.